graphql/schema/types/file.graphql

```diff
@@ -0,0 +1,97 @@
+type Fingerprint {
+    type: String!
+    value: String!
+}
+
+type Folder {
+    id: ID!
+    path: String!
+
+    parent_folder_id: ID
+    zip_file_id: ID
+
+    mod_time: Time!
+
+    created_at: Time!
+    updated_at: Time!
+}
+
+interface BaseFile {
+    id: ID!
+    path: String!
+    basename: String!
+
+    parent_folder_id: ID!
+    zip_file_id: ID
+
+    mod_time: Time!
+    size: Int!
```

Comment:
Won't `Int` cause issues for 32bit systems?( arm32 releases)
Any `size` should be `Int64` (not sure if thats supported from the frontend's library)
I think we used `String` till now to avoid the issue

Reply:
I have now introduced an `Int64` scalar that marshals to and from a string.